pkg/controller/kibana/config_settings_test.go

```diff
@@ -346,9 +379,18 @@ func TestNewConfigSettingsExistingEncryptionKey(t *testing.T) {
 	require.NoError(t, err)
 	var gotCfg map[string]interface{}
 	require.NoError(t, got.Unpack(&gotCfg))
+
```

Comment:
I wonder if it would make the test easier to read if we were to unpack into `reusableSettings`